tests/test_reckoner.py

```diff
@@ -342,61 +343,66 @@ def test_chart_install(self):
         for chart in self.charts:
             self.subprocess_mock.assert_called()
             os.environ[test_environ_var_name] = test_environ_var
-            chart.install(test_namespace)
+            assert chart.install(test_namespace) == None
             logging.debug(chart)
 
-            last_mock = self.subprocess_mock.call_args_list[-1][0][0]
-            self.assertEqual(
-                last_mock[0:6],
-                ['helm', 'upgrade', '--install', chart.release_name, chart.chart_path,
-                 '--namespace={}'.format(chart.namespace)]
-            )
-            if chart.name == test_environ_var_chart:
-                self.assertEqual(
-                    last_mock,
-                    ['helm', 'upgrade', '--install', chart.release_name, chart.chart_path,
-                     '--namespace={}'.format(chart.namespace),
-                     '--recreate-pods',
-                     '--set={}={}'.format(test_environ_var_name, test_environ_var)]
-                )
-            if chart.release_name == test_values_strings_chart:
-                self.assertEqual(
-                    last_mock,
-                    [
-                        'helm', 'upgrade', '--install',
-                        chart.release_name,
-                        chart.chart_path,
-                        '--namespace={}'.format(chart.namespace),
-                        '--recreate-pods',
-                        '--version=0.1.0',
-                        '--set-string=string=string',
-                        '--set-string=integer=10',
-                        '--set-string=boolean=True'
-                    ]
-                )
+            # TODO - we really need to refactor this to be better about testing in the same layer
+            #        this is traversing many layers in the code that could be better encapsulated
+            # COMMENTED OUT NICK HUANCA 2018-
```

Comment:
Are you planning on refactoring this as part of this PR?

Reply:
I don't think so - it would be too large of a change to refactor if i'm not really changing too much here I wanted to keep all these tests as similar as possible.

Reply:
I agree that it would make it too large.
Is it worth leaving these tests in until you are ready to refactor them though? I'm not arguing that they don't need fixing, but they do currently serve to test the process in some way do they not?

Reply:
I think they provide some value for sure. Even though it can be difficult to augment them right now, they provide a useful guide for what's valuable to test for functionality.